detoks-uvt/src/main/java/nl/tudelft/trustchain/detoks/community/UpvoteCommunity.kt

```diff
@@ -44,24 +44,28 @@ class UpvoteCommunity(
     var seedVideoIDs = mutableListOf<Pair<String, String>>()
     var failedSeeds = mutableListOf<Pair<String, String>>()
 
+    private var upvoteService: UpvoteService
+    private var seedRewardService: SeedRewardService
+
     init {
-        messageHandlers[MessageID.UPVOTE_TOKEN] = ::onUpvoteTokenPacket
         messageHandlers[MessageID.MAGNET_URI_AND_HASH] = ::onMagnetURIPacket
         messageHandlers[MessageID.UPVOTE_VIDEO] = ::onUpvoteVideoPacket
         messageHandlers[MessageID.SEED_REWARD] = ::onSeedRewardPacket
-        messageHandlers[MessageID.UPVOTE_TOKEN] = ::onUpvoteTokenPacket
-        this.registerBlockSigner(UpvoteTrustchainConstants.BALANCE_CHECKPOINT, object : BlockSigner {
+        this.registerBlockSigner(UpvoteTrustChainConstants.BALANCE_CHECKPOINT, object : BlockSigner {
             override fun onSignatureRequest(block: TrustChainBlock) {
                 this@UpvoteCommunity.createAgreementBlock(block, mapOf<Any?, Any?>())
             }
         })
         messageHandlers[MessageID.RECOMMENDATION_REQUEST] = ::onRecommendationRequestPacket
         messageHandlers[MessageID.RECOMMENDATION_RECEIVED] = ::onRecommendationReceivedPacket
         messageHandlers.entries.forEach { Log.i("Detoks", "key is ${it.key} and function is ${it.value.toString()}") }
+
+        upvoteService = UpvoteService(OwnedTokenManager(context), SentTokenManager(context))
+        seedRewardService = SeedRewardService(OwnedTokenManager(context))
+
     }
 
     object MessageID {
-        const val UPVOTE_TOKEN = 8
```

Comment:
why is this messageID removed?

Reply:
As we use Upvote_Video now. Upvote_Token used only 1 token, but we send multiple tokens over the network now